internal_docs/review-apps.md

```diff
@@ -0,0 +1,18 @@
+# Pull Request Review apps
+
+This repo uses [Heroku review apps](https://devcenter.heroku.com/articles/github-integration-review-apps)
+to give reviewers a way to easily preview and test changes to the app.
+
+Previews of Pull Requests are automatically published to a URL which has the
+prefix `govuk-prototype-kit` followed by the identifier number of the pull request.
+
+For example, pull request #137 would be deployed to `govuk-prototype-kit-pr-137.herokuapp.com`.
+
+The govuk-design-system-ci user should comment on each PR shortly after
+building the review app with a link.
```

Comment:
Unless something has changed, I think this is specific to the `govuk-design-system` repo, which uses Netlify rather than Heroku.

With Heroku, review apps will be associated as 'deployments' of the PR. There won't be a comment, but deploys will appear in the timeline as 'events' and the deployment will be listed at the bottom of the PR above the status checks.

Reply:
Ah, I wasn't sure what the little status thingies were called. I'll update the doc to reflect this.

Reply:
Updated, let me know if the new content makes sense to you.